app/src/main/java/agency/five/codebase/android/movieapp/ui/component/MovieCard.kt

```diff
@@ -0,0 +1,54 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import agency.five.codebase.android.movieapp.mock.MoviesMock
+import agency.five.codebase.android.movieapp.model.Movie
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+
+@Composable
+fun MovieCard(
+    modifier: Modifier = Modifier,
+    movie: Movie,
+    onMovieClick: () -> Unit = {}
+) {
+    Card(
+        modifier = modifier
+            .clickable { onMovieClick() }
+            .padding(10.dp)
+            .size(8.dp),
```

Comment:
You should remove this because you won't be able to change size of this component when you'll be using it in next homework. Generally, in order to make component reusable, you shouldn't hardcode its size.